app/main/util/render_utils.py

```diff
@@ -128,11 +286,26 @@ def generate_image_manifest(
     image = Image.open(io.BytesIO(s3_file_object["Body"].read()))
     image_width, image_height = image.size
 
+    # Detect image format
+    image_format = image.format.lower() if image.format else "png"
+    if image_format == "jpeg":
+        mime_type = "image/jpeg"
+    elif image_format == "png":
+        mime_type = "image/png"
+    elif image_format in ["tiff", "tif"]:
+        mime_type = "image/tiff"
+    elif image_format == "gif":
+        mime_type = "image/gif"
+    elif image_format == "webp":
+        mime_type = "image/webp"
+    else:
+        mime_type = f"image/{image_format}"
+
     manifest = {
-        "@context": "https://iiif.io/api/presentation/3/context.json",
+        "@context": "http://iiif.io/api/presentation/3/context.json",
```

Comment:
Minor but this redirects to https so might as well use the https version
```suggestion
        "@context": "https://iiif.io/api/presentation/3/context.json",
```

Reply:
This change has been made